.github/workflows/pull-request.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       # Check the Quality Gate status.
       - name: SonarQube Quality Gate check
         id: sonarqube-quality-gate-check
-        uses: sonarsource/sonarqube-quality-gate-action@master
+        uses: sonarsource/sonarqube-quality-gate-action@72f24ebf1f81eda168a979ce14b8203273b7c3ad # pin@master
```

Comment:
Should we use an actual version of the workflow here @mthkbarber?

Reply:
I think we would still want a change here, just pinning to a version like the other actions 🙂

Reply:
Lets try again ^_^